### PR TITLE
Disclaimer

### PR DESCRIPTION
**Pull request by sushi2k**
We have no disclaimer yet, this is a first draft to create one.

### CODE REVIEW

**Comment by OMTGreleaser** [comment]
Results for commit 6c4f314f1991d7973b8f7786cbf4d322560e511c: Broken link result: [!] Error(s) found by the Links validator: 8 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        4 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by OMTGreleaser** [comment]
Results for commit 24558f1d1320e44b32076ea67179a23c2cbfa49a: Broken link result: [!] Error(s) found by the Links validator: 7 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        5 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -29,6 +29,18 @@ Project maintainers are responsible for clarifying the standards of acceptable b
...
+
+The MSTG and it's materials are for educational and research purposes only, or if you are a professional security tester that is analysing mobile apps with approval of the app owner (e.g. Penetration Test or Bug Bounty Program).
+
+Please consult the laws in your country before executing any tests against mobile apps by utilizing the MSTG materials. Refrain from violating the laws with anything described in the MSTG.
+
+The MSTG project is also NOT supporting in any way of unethical activities and if you come across such requests please reach out to the project leaders and raise this to us.
+
+Neither OWASP, project leaders, authors or anyone else involved in this project is going to accept responsibility for your actions.
+
+The intention of the MSTG is not to encourage hacking or malicious activities! Instead, the goal of the project is to document different attacks against mobile apps and it's risk and offer ways to reduce or mitigate it's risk.
```
Comment (cpholguera) [inline CODE_OF_CONDUCT.md]:
```suggestion
The intention of the MSTG is not to encourage hacking or malicious activities! Instead, the goal of the project is to document different attacks against mobile apps and its risk and offer ways to reduce or mitigate that risk.
```

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -29,6 +29,18 @@ Project maintainers are responsible for clarifying the standards of acceptable b
...
 
+## Disclaimer
+
+The MSTG and it's materials are for educational and research purposes only, or if you are a professional security tester that is analysing mobile apps with approval of the app owner (e.g. Penetration Test or Bug Bounty Program).
+
+Please consult the laws in your country before executing any tests against mobile apps by utilizing the MSTG materials. Refrain from violating the laws with anything described in the MSTG.
+
+The MSTG project is also NOT supporting in any way of unethical activities and if you come across such requests please reach out to the project leaders and raise this to us.
+
+Neither OWASP, project leaders, authors or anyone else involved in this project is going to accept responsibility for your actions.
```
Comment (cpholguera) [inline CODE_OF_CONDUCT.md]:
```suggestion
Neither OWASP, the MSTG project leaders, authors or anyone else involved in this project is going to accept responsibility for your actions.
```

Comment (cpholguera) [reply CODE_OF_CONDUCT.md]: Apart from that: Accept or take responsibility?

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -29,6 +29,18 @@ Project maintainers are responsible for clarifying the standards of acceptable b
 
 Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct, or to ban temporarily or permanently any contributor for other behaviors that they deem inappropriate, threatening, offensive, or harmful.
 
+## Disclaimer
+
+The MSTG and it's materials are for educational and research purposes only, or if you are a professional security tester that is analysing mobile apps with approval of the app owner (e.g. Penetration Test or Bug Bounty Program).
+
+Please consult the laws in your country before executing any tests against mobile apps by utilizing the MSTG materials. Refrain from violating the laws with anything described in the MSTG.
+
+The MSTG project is also NOT supporting in any way of unethical activities and if you come across such requests please reach out to the project leaders and raise this to us.
```
Comment (cpholguera) [inline CODE_OF_CONDUCT.md]:
```suggestion
The MSTG project is also NOT supporting in any way of unethical activities and if you come across such requests please reach out to the project leaders and raise this to them.
```

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -29,6 +29,18 @@ Project maintainers are responsible for clarifying the standards of acceptable b
 
 Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct, or to ban temporarily or permanently any contributor for other behaviors that they deem inappropriate, threatening, offensive, or harmful.
 
+## Disclaimer
+
+The MSTG and it's materials are for educational and research purposes only, or if you are a professional security tester that is analysing mobile apps with approval of the app owner (e.g. Penetration Test or Bug Bounty Program).
```
Comment (cpholguera) [inline CODE_OF_CONDUCT.md]:
```suggestion
The MSTG and its materials are conceived for educational and research purposes only, or if you are a professional security tester that is analyzing mobile apps with approval of the app owner (e.g. as part of a penetration test or a bug bounty program).
```

File: CODE_OF_CONDUCT.md [hunk 5]
```diff
@@ -29,6 +29,18 @@ Project maintainers are responsible for clarifying the standards of acceptable b
...
+
+Please consult the laws in your country before executing any tests against mobile apps by utilizing the MSTG materials. Refrain from violating the laws with anything described in the MSTG.
```
Comment (cpholguera) [inline CODE_OF_CONDUCT.md]: What do you think of this rephrasing?
```suggestion
Refrain from violating the laws in your country by carefully consulting them before executing any tests against mobile apps by utilizing the MSTG materials.
```

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (sushi2k) [reply CODE_OF_CONDUCT.md]: I made it now "take responsibility". Have a look

**Comment by sushi2k** [comment]
Thanks @cpholguera, now it reads smoother! Have a look again

**Comment by OMTGreleaser** [comment]
Results for commit 8acb9f71e39bbde9a73dd18ff2c724fcd9395886: Broken link result: [!] Error(s) found by the Links validator: 6 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        5 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

File: CODE_OF_CONDUCT.md [hunk 6]
```diff
@@ -29,6 +29,18 @@ Project maintainers are responsible for clarifying the standards of acceptable b
...
+
+The MSTG and its materials are conceived for educational and research purposes only, or if you are a professional security tester that is analyzing mobile apps with approval of the app owner (e.g. as part of a penetration test or a bug bounty program).
+
+Refrain from violating the laws in your country by carefully consulting them before executing any tests against mobile apps by utilizing the MSTG materials.
+
+The MSTG project is also NOT supporting in any way of unethical activities and if you come across such requests please reach out to the project leaders and raise this to them.
+
+Neither OWASP, the MSTG project leaders, authors or anyone else involved in this project is going to take responsibility for your actions.
+
+The intention of the MSTG is not to encourage hacking or malicious activities! Instead, the goal of the project is to document different attacks against mobile apps and its risk and offer ways to reduce or mitigate that risk.
```
Comment (cpholguera) [inline CODE_OF_CONDUCT.md]: Maybe better with the comma?
```suggestion
The intention of the MSTG is not to encourage hacking or malicious activities! Instead, the goal of the project is to document different attacks against mobile apps and its risk, and offer ways to reduce or mitigate that risk.
```

**Comment by OMTGreleaser** [comment]
Results for commit db3597dcfe90cdad45ce5997c53e648d3d7e6536: Broken link result: [!] Error(s) found by the Links validator: 6 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        5 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

File: CODE_OF_CONDUCT.md [hunk 7]
```diff
@@ -29,6 +29,18 @@ Project maintainers are responsible for clarifying the standards of acceptable b
 
 Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct, or to ban temporarily or permanently any contributor for other behaviors that they deem inappropriate, threatening, offensive, or harmful.
 
+## Disclaimer
+
+The MSTG and its materials are conceived for educational and research purposes only, or if you are a professional security tester that is analyzing mobile apps with approval of the app owner (e.g. as part of a penetration test or a bug bounty program).
+
+Refrain from violating the laws in your country by carefully consulting them before executing any tests against mobile apps by utilizing the MSTG materials.
+
+The MSTG project is also NOT supporting in any way of unethical activities and if you come across such requests please reach out to the project leaders and raise this to them.
```
Comment (cpholguera) [inline CODE_OF_CONDUCT.md]:
```suggestion
The MSTG project is also NOT supporting unethical activities in any way. If you come across such requests, please reach out to the project leaders and raise this to them.
```

File: CODE_OF_CONDUCT.md [hunk 8]
```diff
@@ -29,6 +29,18 @@ Project maintainers are responsible for clarifying the standards of acceptable b
 
 Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct, or to ban temporarily or permanently any contributor for other behaviors that they deem inappropriate, threatening, offensive, or harmful.
 
+## Disclaimer
+
+The MSTG and its materials are conceived for educational and research purposes only, or if you are a professional security tester that is analyzing mobile apps with approval of the app owner (e.g. as part of a penetration test or a bug bounty program).
+
+Refrain from violating the laws in your country by carefully consulting them before executing any tests against mobile apps by utilizing the MSTG materials.
```
Comment (cpholguera) [inline CODE_OF_CONDUCT.md]: What about "when" not to repeat "by"?

```suggestion
Refrain from violating the laws in your country by carefully consulting them before executing any tests against mobile apps when utilizing the MSTG materials.
```

**Comment by sushi2k** [comment]
Agree. Just merged them :-)

**Comment by OMTGreleaser** [comment]
Results for commit e2d05a932027a9e9a4336ee1a29cb584f53b203f: Broken link result: [!] Error(s) found by the Links validator: 8 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        5 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by cpholguera** [comment]
Thank you Sven! Line 38 is still open, right? Once fixed we can merge ;) if you have another suggestion for it please let me know ☺️

**Comment by sushi2k** [comment]
Ups. Must have missed to press the button, defiantly read it and agreed to it 😊

**Comment by OMTGreleaser** [comment]
Results for commit 784565a1850e79eb683b5ca79d70013585b3f633: Broken link result: [!] Error(s) found by the Links validator: 7 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        5 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Pull request merged**